scripts/mutect2_wdl/mutect2-replicate-validation.wdl

```diff
@@ -76,10 +76,8 @@ workflow Mutect2ReplicateValidation {
 	File ref_dict
 	File? pon
 	File? pon_index
-	File? dbsnp
-	File? dbsnp_index
-	File? cosmic
-	File? cosmic_index
+	File? gnomad
```

Comment:
Just a reminder that we will have to provide whatever we need for these gnomAD parameters available for download outside the Broad and for the sequencing platform.

I'm assuming that whatever you are putting in this parameter is available for public consumption.